pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -345,7 +345,7 @@ func applyTrafficPolicy(env *model.Environment, cluster *v2.Cluster, policy *net
 	applyConnectionPool(cluster, connectionPool)
 	applyOutlierDetection(cluster, outlierDetection)
 	applyLoadBalancer(cluster, loadBalancer)
-	applyUpstreamTLSSettings(cluster, tls, env.Mesh.SdsUdsPath)
+	applyUpstreamTLSSettings(cluster, tls, env.Mesh.SdsUdsPath, env.Mesh.EnableSdsTokenMount)
```

Comment:
please pass env here. no need to pass every param

Reply:
updated.